.pipelines/custom-templates/publish-template.yml

```diff
@@ -98,7 +98,7 @@ jobs:
             inputs:
                 command: "custom"
                 workingDir: "${{ parameters.path }}/${{ parameters.libName }}"
-                customCommand: "run deploy:beta" # Remove :beta when ready to GA"
```

Comment:
Should we remove this script from the package.json as well?

Reply:
We can, I assume we'll use it for future beta releases though?